documentation/src/main/docs/faq.adoc

```diff
@@ -323,6 +323,16 @@ Here we pass the default worker thread pool, but you can use your own executor.
 Note that `subscribeOn` does not subscribe to the `Uni`.
 It specifies the `executor` to use when a subscribe call happens.
 
+Using `subscribeOn` works when the blocking operation happens at subscription time.
+But, we you are dealing with `Multi` and need to execute a blocking operation for each item, you need `emitOn`.
```

Comment:
But, when you are dealing with `Multi` and need to execute a blocking operation for each item, you need `emitOn`.

Reply:
Thanks! Fixed.